common/src/main/java/net/draycia/carbon/common/CarbonChatCommon.java

```diff
@@ -38,6 +43,24 @@ public final void initialize() {
             }));
     }
 
+    @Provides
+    @Singleton
+    public CarbonMessageService messageService(
```

Comment:
Reminder to move this out of the CarbonChatCommon class and into something more suitable for providers